src/test/java/com/redislabs/redisai/RedisAITest.java

```diff
@@ -6,129 +6,327 @@
 import redis.clients.jedis.Jedis;
 import redis.clients.jedis.JedisPool;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.Map;
 
 public class RedisAITest {
 
-  private final JedisPool pool = new JedisPool();
-  private final RedisAI client = new RedisAI(pool); 
-  
-  @Before
-  public void testClient() {
-    try (Jedis conn = pool.getResource()) {
-      conn.flushAll();
-    }      
-  }
-  
-  @Test
-  public void testSetTensor() {
-    Assert.assertTrue(client.setTensor("t1", new float[][] {{1,2},{3,4}}, new int[] {2,2}));
-//    client.getTensor("t1");
-  }
-
-  @Test
-  public void testSetModel() {
-    ClassLoader classLoader = getClass().getClassLoader();
-    String model = classLoader.getResource("graph.pb").getFile();
-    Assert.assertTrue(client.setModel("model", Backend.TF, Device.CPU, new String[] {"a", "b"}, new String[] {"mul"}, model));
-//    client.getModel("model");
-  }
-
-  @Test
-  public void testRunModel() {
-    ClassLoader classLoader = getClass().getClassLoader();
-    String model = classLoader.getResource("graph.pb").getFile();
-    client.setModel("model", Backend.TF, Device.CPU, new String[] {"a", "b"}, new String[] {"mul"}, model);
-    
-    client.setTensor("a", new float[] {2, 3}, new int[]{2});
-    client.setTensor("b", new float[] {3, 5}, new int[]{2});
-
-    Assert.assertTrue(client.runModel("model", new String[] {"a", "b"}, new String[] {"c"}));
-    Tensor tensor = client.getTensor("c");
-    float[] values = (float[]) tensor.getValues();
-    float[] expected =  new float[] {6, 15};
-    Assert.assertTrue("Assert same shape of values", values.length==2);
-    Assert.assertArrayEquals(values,expected, (float) 0.1);
-  }
-
-  @Test
-  public void testSeScriptFile() {
-    ClassLoader classLoader = getClass().getClassLoader();
-    String scriptFile = classLoader.getResource("script.txt").getFile();
-    Assert.assertTrue(client.setScriptFile("script", Device.CPU, scriptFile));
-  }
-  
-  @Test
-  public void testSeScript() {
-    String script = "def bar(a, b):\n" + 
-    "    return a + b\n";
-    Assert.assertTrue(client.setScript("script", Device.CPU, script));
+    private final JedisPool pool = new JedisPool();
+    private final RedisAI client = new RedisAI(pool);
+    private final RedisAI client_default_connection = new RedisAI();
+    private final RedisAI client_host_port = new RedisAI("localhost", 6379);
```

Comment:
What are these connections for?
```
    private final RedisAI client_default_connection = new RedisAI();
    private final RedisAI client_host_port = new RedisAI("localhost", 6379);
    private final RedisAI client_poolsize = new RedisAI("localhost", 6379, 30000, 1);
    private final RedisAI client_password = new RedisAI("localhost", 6379, 30000, 1, "");
```

Reply:
will add simple tests to it. Want all connection modes to be tests.

Reply:
fixed on the new commit